lib/faraday/encoders/flat_params_encoder.rb

```diff
@@ -29,7 +29,6 @@ def self.encode(params)
       buffer = +''
       params.each do |key, value|
         encoded_key = escape(key)
-        value = value.to_s if value == true || value == false
```

Comment:
This is handled by `Faraday::Utils.escape` already:

```ruby
Faraday::Utils.escape(true)
#=> "true"
```

Reply:
I got curious about the test for this, where a `value` is a Boolean:

`.encode([{ 'a' => true, 'b' => false}])`

Perhaps we want that test, too?

Reply:
Agree with @olleolleolle, we need a test that tests `encode` method with boolean values, if there isn't one already.

Reply:
I might have been a bit too quick. Will have a look at it tonight. 🙂

Reply:
After adding some more tests, it looks like this line really is redundant.